src/filter_noise.py

```diff
@@ -0,0 +1,35 @@
+from core_data_modules.logging import Logger
+from src.lib.pipeline_configuration import PipelineConfiguration
+
+log = Logger(__name__)
+
+
+class FilterNoise(object):
+    @staticmethod
+    def filter_noise_radio_messages(messages):
+        """
+        Filters a list of messages which have been labelled as Noise_Other_Project.
+
+        :param messages: List of message objects to filter.
+        :type messages: list of TracedData
+        :param message_keys: Keys in each TracedData to search for a message.
```

Comment:
This argument isn't actually in the signature

Reply:
Removed thanks